IK-1429: feat: Introduced 'mlSettings' parameter in plugin options, Typescript support (#14)
imagekit-developer/embeddable-media-library: README.md

  },
  view: 'modal',  // inline | modal (default)
  renderOpenButton: true,  // false | true (default)

>   /*
>   mlSettings: {  // optional
>   // sets initial state of Media Library, check the document link in 'Usage' section for more information
>     initialView: { 
>       folderPath: "<your-folder-path>"
>     },
>     multiple: true // false | true (default)
>     maxFiles: 20 // relevant when `multiple` is true
>   }
>   */

Reviewer: ```suggestion
  /*
  mlSettings: {  // optional, sets initial state of Media Library, refer to the 'Usage' section for more information
    initialView: { 
      folderPath: "<your-folder-path>"
    },
    multiple: true // false | true (default)
    maxFiles: 20 // relevant when `multiple` is true
  }
  */
```
